### PR TITLE
openwisp-config: update to 1.0.0

### PR DESCRIPTION
**Pull request by nemesisdesign**
Maintainer: @nemesisdesign 
Compile tested: ath79, ramips
Run tested: ath79, ramips

### CODE REVIEW

**Review by nemesisdesign — COMMENTED** [review]
@neheb @feckert any problem with this? Can we get it merged?

File: admin/openwisp-config/Makefile [hunk 1]
```diff
@@ -6,15 +6,16 @@ include $(TOPDIR)/rules.mk
...
 PKG_MAINTAINER:=Federico Capoano <f.capoano@openwisp.io>
 PKG_LICENSE:=GPL3.0-or-later
 
 PKG_SOURCE_URL:=https://github.com/openwisp/openwisp-config.git
-PKG_MIRROR_HASH:=b1f5682715cc5b5623b2acf49ebb75d8bac1b58094981b61ede3ca392bdfa269
+PKG_MIRROR_HASH:=f0b0866b27a7a75bc33c3bbc4780dd99f5ae693226012ae04e63a303172c2c38
 PKG_SOURCE_PROTO:=git
-PKG_SOURCE_DATE:=2021-12-07
-PKG_SOURCE_VERSION:=08df34db29180a0d13c2e357723e8c546523014a
+PKG_SOURCE_DATE:=2022-05-10
```
Comment (neheb) [inline admin/openwisp-config/Makefile:17]: There's no need in PKG_SOURCE_DATE if there's a proper version.

File: admin/openwisp-config/Makefile [hunk 2]
```diff
@@ -6,15 +6,16 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=openwisp-config
 PKG_RELEASE:=1
+PKG_VERSION=1.0.0
```
Comment (neheb) [inline admin/openwisp-config/Makefile:9]: this gets generated from PKG_SOURCE_VERSION. No need for it.

**Comment by nemesisdesign** [comment]
Thanks for the review @neheb, I have updated it.

**Comment by feckert** [comment]
LGTM Thanks

**Comment by nemesisdesign** [comment]
Thanks :+1:

**Pull request merged**